modules/core/src/main/java/org/apache/synapse/mediators/MediatorLog.java

```diff
@@ -114,8 +121,10 @@ public void auditLog(Object msg) {
      * Log a message at level DEBUG to all available/enabled logs.
      */
     public void auditDebug(Object msg) {
-        defaultLog.debug(msg);
-        if (synCtx.getServiceLog() != null) {
+    	if (defaultLog.isDebugEnabled()) { 
```

Comment:
This if condition is not required, isDebugEnabled is checked inside the debug() method.